tf2onnx/tfonnx.py

```diff
@@ -2229,6 +2231,28 @@ def rewrite_conv2d_with_pad(g, ops):
     return ops
 
 
+def rewrite_const_sub_with_pack(g, ops):
```

Comment:
why not enable rewrite_constant_fold ?

Reply:
@zhijxu-MS, let's withdraw rewrite_const_sub_with_pack from this PR to unblock onehot.
const fold can be in another pr.

Reply:
@guschmue there is a bug in rewrite_constant_fold when I tried with a real model and after discussion with @nbcsm we decided to enhance const fold in another way.

I will withdraw my changes in this PR, and send the enhancement in another PR.

Reply:
Works for me.  I think fixing rewrite_constant_fold is the better option so we have all const fold in one place. rewrite_constant_fold used to be needed for older tensorflow versions (for example to get bert work) but since 1.12 I have not seen issues.

Reply:
https://github.com/onnx/tensorflow-onnx/issues/417